docs/source/references/routes.rst

```diff
@@ -1,53 +1,12 @@
 Routes
 ======
 
-Support for relationships
--------------------------
-
-Relationships must be explicitly defined in the Data Resource Schema document.
-
-Supported relationships
-^^^^^^^^^^^^^^^^^^^^^^^
-
-The following list of relationships are supported:
-
-- One to Many (not self join)
-- Many to Many
-
-The following are not currently supported:
-
-- One to Many (self join)
-
-One to Many support
-^^^^^^^^^^^^^^^^^^^
-
-You may define one to many relationships using tableschema foregin key in the Data Resource Schema document. Please see `tableschema's foreign key documentation <https://specs.frictionlessdata.io/table-schema/#foreign-keys>` for more information.
-
-Once defined you may access the relationship as a field on your resource.
-
-Standard relational database referential integrity still applies:
-
-- If you reference a resource it must already exist or else the application will return an error.
-
-Many to Many support
-^^^^^^^^^^^^^^^^^^^^
-
-There are limitations to many to many support:
-
-- Both tables must contain a single primary key that is an integer.
-
-The application will handle creating an association table.
-
-An error will return if either the primary key IDs provided for the parent and children fail to resolve to existing data in the database.
-
-In order for many to many to work both primary keys of the tables in the relationship need to "id".
-
-Generated routes and how to interact with them
+Generated Routes and how to interact with them
```

Comment:
What do you think about calling this "Interacting with Generated Data Resources"? And then, you could link to this page from the "Steps to Use the DRG" section.